### DISCUSSION TITLE
ctags in nvchad

### DISCUSSION THREAD

**Discussion by Scythe66666** [NvChad/NvChad#2567] [Q&A]
how to make my nvchad look into tags file generated by ctags for tags instead of it look in active buffers only

**Comment by siduck** [comment]
tags file? idk whats that

**Reply by Scythe66666** [reply]
tags file the file that is generated when you do ctags -R *. Basically i am not able to jump to function definition in c language properly. I am using clangd and i am having multiple different client offset encodings detected for buffer this is not supported yet for ctags

**Reply by siduck** [reply]
@Scythe66666 the clang issue is pretty famous, just google it and fix
